internal/service/onlinearchive/data_source_online_archive.go

```diff
@@ -216,12 +216,12 @@ func schemaOnlineArchive() map[string]*schema.Schema {
 }
 
 func dataSourceMongoDBAtlasOnlineArchiveRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
-	connV2 := meta.(*config.MongoDBClient).AtlasV2
+	connOldV2 := meta.(*config.MongoDBClient).OldAtlasV2
```

Comment:
Could we use the name `v20231001002AtlasSDK` instead of `OldAtlasV2`? It is not easy to understand what we consider the old atlas v2

Reply:
I'm following the convention @oarbusi did in his previous PR, e.g. it's being used in federated_settings_identity_provider service

Reply:
I missed that PR 😭  What is our opinion on the name used for the Atlas SDK?

Reply:
I think the correct naming would include the dates to avoid confusion if more parallel sdks are mantained

Reply:
right now I think it's a premature optimization as we only have 2 sdks, y we intend to have the old one a short period of time.  if more than 2 are would be needed at some moment then I agree.

Reply:
at the end I've added the date to the old sdk as per your recommendation